src/Language/Haskell/Liquid/UX/CmdLine.hs

```diff
@@ -743,15 +743,15 @@ reportResult logResultFull cfg targets out = do
          let outputResult = resDocs tidy cr
          -- For now, always print the \"header\" with colours, irrespective to the logger
          -- passed as input.
-         liftIO $ printHeader (colorResult r) (orHeader outputResult)
+         -- liftIO $ printHeader (colorResult r) (orHeader outputResult)
```

Comment:
This is the `**** LIQUID SAFE ***********` header.  I removed it because it made `-j` easier to do, but see discussion in #1955 for whether we should keep it.